app/cookies.ts

```diff
@@ -0,0 +1,18 @@
+import { createCookie } from "remix";
```

Comment:
@frederike-ramin I just discovered the `createCookieSessionStorage` in the docs: https://remix.run/docs/en/v1/api/remix#createcookiesessionstorage Have you seen it, is it useful here? It's also fine to not go further here, as we don't want to use cookies for storage anyway.

Reply:
Ah, good to know. I didn't see that before. However, I think I'd rather not go into it too much to not put more effort into the cookie storage (so that I don't get too attached to it. :D)